Database/Stored Procedures/Add_or_Update/add_or_update_podium.sql

```diff
@@ -35,6 +35,9 @@ BEGIN
 		/* Checking the constraints on performance */
 		EXEC sp_check_podium @tent_number, @construction_width, @construction_length, @construction_height, @floor_height, @capacity, @free_span_width, @free_span_length, @free_span_height
 
+		/* Check if the podium and tent are from the same festival */
+		EXEC sp_check_podium_tent_festival @podium_number, @tent_number
+
```

Comment:
Jij hebt hem voor het laatst geëdit. Het staat alleen nog niet in de banner bovenaan.